lib/KTable/useSorting/index.js

```diff
@@ -24,25 +24,39 @@ export default function useSorting(headers, rows, defaultSort, disableBuiltinSor
   const sortKey = ref(null);
   const sortOrder = ref(null);
 
+  /**
+   * Helper function to get the value for sorting.
+   * @param {Object} row - The row object.
+   * @param {Number} index - The index of the column to sort by.
+   * @returns {any} - The value to be used for sorting.
+   */
+  const getSortValue = (row, index) => {
+    const value = row[index];
+    if (typeof value === 'string') {
+      return value.toString().toLocaleLowerCase();
```

Comment:
One minor issue- Adding a check to ensure that the value is not null or undefined before attempting to convert it to lowercase, this would prevent potential errors if the data has missing values.

Reply:
Added check for null values.